### DISCUSSION TITLE
Thinking of breaking the fork from mpenning?

### DISCUSSION THREAD

**Discussion by falkowich** [falkowich/ise#161] [General]
Hi,

I am thinking of breaking the backward fork link from https://github.com/mpenning/ise.
Today things went somewhat mad with a merge and I had some problems that it was a fork with really old updates.

In the readme all credits will still be made to the creators:
> All initial work is done by https://github.com/bobthebutcher and https://github.com/bobthebutcher
> I forked from them and updated so it worked with ISE 2.2.x and changed all functions to json calls.

Any thoughts out there?
I hope that this is ok for @bobthebutcher and @mpenning?

--
Regards Falk

**Comment by mpenning** [comment]
Hello Falk, I have no concerns
